csv/result.go

```diff
@@ -579,10 +579,10 @@ DONE:
 }
 
 func (d *tableDecoder) init(line []string) error {
-	if d.meta.TableID != "" {
-		d.id = d.meta.TableID
-	} else if len(line) != 0 {
+	if len(line) != 0 {
```

Comment:
I'm unclear on what `line` is in this context...Is it the `#defaults` annotation row, or the first line of the table, or something else?

Reply:
@wolffcm `line` represents the first record from the table.

Reply:
It would be great to rename this parameter to something like `firstRecord`.